### ISSUE TITLE
Output

### ISSUE THREAD

**Issue by tanmayd26** [pengzhi1998/Model_based_active_mapping#2]
I am new to reinforcement learning and I am trying to run the files. I get this as output which is not same as yours in the repo.

![Figure_1](https://github.com/user-attachments/assets/b2f3bacf-8808-4267-b6bf-77a056db5db3)

**Comment by pengzhi1998** [comment]
Hi, thank you for your interest to our work.

I didn't fully get your question, which figure in our repo are you comparing to? Actually, the trajectory the agent chooses highly depends on the relative positions of the landmarks.

**Comment by tanmayd26** [comment]
https://github.com/pengzhi1998/Model_based_active_mapping/blob/main/3landmarks.gif?raw=true

I am comparing to this. I followed all the steps mentioned in the repo.

**Comment by pengzhi1998** [comment]
The figure you chose from our repo has only 3 landmarks, while the figure you showed above has 5 landmarks and they seem to be static. These differences would lead to a big difference in the agent's trajectory.

**Comment by tanmayd26** [comment]
Okay, let me rephrase my question. How can I get the below behavior?

![3landmarks](https://github.com/user-attachments/assets/238b4b58-3b62-4d2f-86ab-60dd28db02d1)

**Comment by pengzhi1998** [comment]
You could take a look at our branch: https://github.com/pengzhi1998/Model_based_active_mapping/tree/model_based_compare, which compares the agent's behavior with model-based and model-free RL.
